package.json

```diff
@@ -36,8 +39,10 @@
     "tape-run": "^9.0.0"
   },
   "devDependencies": {
+    "dependency-check": "^5.0.0-4",
     "eslint": "^7.32.0",
-    "eslint-config-standard": "^16.0.3",
-    "npm-run-all": "^4.1.5"
+    "eslint-config-standard-with-typescript": "^21.0.1",
+    "npm-run-all": "^4.1.5",
+    "typescript": "^4.5.5"
```

Comment:
There will be a warning when running eslint on the TS code saying that we're using too new of a TS version. This is because TS 4.5 isn't officially supported until ESLint 8. However, `standard` doesn't support ESLint 8 yet so we'll have to wait on them to upgrade (should happen in the next months). Regardless, the warning is harmless until then.